client/my-sites/plan-features/list.jsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+class PlanFeaturesItemList extends Component {
+
+	render() {
+		return (
+			<ul className="plan-features__item-list">
+				{ this.props.children }
+			</ul>
+		);
+	}
+}
+
+export default PlanFeaturesItemList;
```

Comment:
Just remember that we won't be able to reference stateless components. It's only an issue if we need a popover.